lusSTR/wrappers/filter.py

```diff
@@ -146,9 +146,9 @@ def process_strs(dict_loc, datatype, seq_col, brack_col):
 
 def EFM_output(profile, outfile, profile_type, data_type, col, sex, separate=False):
     if profile_type == "reference":
-        profile = profile[profile.allele_type == "real_allele"]
+        profile = profile.query("allele_type == 'Typed'")
     else:
-        profile = profile[profile.allele_type != "BelowAT"]
+        profile = profile.query("allele_type != ['BelowAT', 'Deleted']")
```

Comment:
Did you make these changes to get rid of the pandas warnings? Because functionally, they look very similar.

I will say though, I was unaware that you could use list syntax in a query string! TIL

Reply:
No, I changed it to use a list since it needs to remove both BelowAT and Deleted sequences. I did it in line 149 just to be consistent in the code.